projects/emissary-ingress/emissary/docker/linux/emissary-image/Dockerfile

```diff
@@ -0,0 +1,61 @@
+ARG BASE_IMAGE 
+ARG BUILDER_IMAGE
+ARG APPMESH_VERSION
+FROM $BUILDER_IMAGE as emissary-builder
+
+COPY emissary/python /buildroot/python
+COPY _output/bin/emissary/linux-amd64 /buildroot/bin
+
+ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin:/buildroot/bin
+
+WORKDIR /buildroot
+
+RUN install_rpm libcap htop && \
+    install_binary /usr/bin/curl /usr/bin/chgrp /usr/bin/chmod && \
+    cleanup "emissary"  && \
```

Comment:
nit: i like this to be at the end of the run block, seems to make more sense since it is the cleanup step.

Reply:
I'll swap it.